imageregistry/v1/types.go

```diff
@@ -306,6 +306,36 @@ type ImageRegistryConfigStorageIBMCOS struct {
 	ServiceInstanceCRN string `json:"serviceInstanceCRN,omitempty"`
 }
 
+// ImageRegistryConfigStorageOSS holds Alibaba Cloud OSS configuration.
+type ImageRegistryConfigStorageOSS struct {
+	// bucket is the bucket name in which you want to store the registry's
+	// data.
+	// Optional, will be generated if not provided.
+	// +optional
+	Bucket string `json:"bucket,omitempty"`
+	// region is the GCS location in which your bucket exists.
+	// Optional, will be set based on the installed GCS Region.
+	// +optional
+	Region string `json:"region,omitempty"`
+	// regionEndpoint is the endpoint for S3 compatible storage services.
+	// Optional, defaults based on the Region that is provided.
+	// +optional
+	RegionEndpoint string `json:"regionEndpoint,omitempty"`
+	// internal specifies whether the registry use the OSS VPC internal endpoint
+	// Optional, defaults to false. if RegionEndpoint is specified, this config will be ignored
```

Comment:
This default surprises me. Why?

Reply:
You mean OSS  VPC internal endpoint are used by default?

Reply:
I think it's better to keep same as registry storage driver  for oss. https://github.com/docker/docker.github.io/blob/master/registry/storage-drivers/oss.md

Reply:
We should be secure by default.  Given a choice of Internal or Public, I think Internal is better.

Reply:
Got it. If user don't set value or the value is out of range, the default value is Internal

var (
   InternalEndpoint = EndpointAccessibility("Internal")
   PublicEndpoint = EndpointAccessibility("Public")
)


```
EndpointAccessibility EndpointAccessibility `json:"endpointAccessibility,omitempty"`
```